samples/snippets/usage_guide.py

```diff
@@ -264,13 +264,19 @@ def _sink_pubsub_setup(client):
     )  # API call
     # [END sink_topic_permissions]
 
-    return topic
+    # create callback wrapper to delete topic when done
+    class TopicDeleter:
+        def delete(self):
+            client.delete_topic(request={"topic": topic_path})
```

Comment:
nit: perhaps, worth adding a log line that topic X is deleted - for easier troubleshooting if tests fail in the future